tests/de/homeassistant_HassGetState.yaml

```diff
@@ -1,2 +1,69 @@
 language: de
-tests: []
+tests:
+  - sentences:
+      - "wie ist die Außentemperatur?"
+    intent:
+      name: HassGetState
+      slots:
+        name: "Außentemperatur"
+    response: "Außentemperatur ist 21 °C"
```

Comment:
this fails, because "somewhere" the `ß` is translated into `ss`, but German umlauts are not affected/translated (_ä ö ü remain as is - manually tested_)

test failure message:
```
E                   AssertionError: Incorrect response for: wie ist die Außentemperatur?
E                   assert 'Aussentemperatur ist 21 °C' in {'Außentemperatur ist 21 °C'}
```

manual sentence test: `$ python3 -m script.intentfest parse --language de --sentence "Wie ist die Außentemperatur?"`
```json
{
  "text": "Wie ist die Außentemperatur?",
  "match": true,
  "intent": "HassGetState",
  "slots": {
    "name": "Außentemperatur"
  },
  "context": {
    "domain": "sensor"
  },
  "response_key": "einzeln",
  "response": "Aussentemperatur ist 21 °C"
}
```

Reply:
also strange for me, maybe we should change it to ss in the test for now and then check again with later versions

Reply:
I've created a new issue for that (#1133) and will change the test here to use `ss`.
As soon as the issue is fixed, this tests will fail and we will get notice about it